src/main/java/org/kairosdb/core/datastore/QueryMetric.java

```diff
@@ -210,7 +210,7 @@ public void setQueryLoggingType(String type) {
 	}
 
 	public String getQueryLoggingType() {
-		return loggingType;
+		return loggingType == null ? "simple" : loggingType;
```

Comment:
Should we do this in the setter method - `setQueryLoggingType `

Reply:
This is needed in cases when setter method was never called and `loggingType` was left `null`. So doing it in the setter won't help